internal/runners/activate/activate.go

```diff
@@ -219,6 +219,9 @@ func (r *Activate) run(params *ActivateParams) error {
 		}
 	}
 
+	// update config with project
```

Comment:
Did you see that we are updating the project mapping whenever we are parsing the project file?
https://github.com/ActiveState/cli/blob/master/pkg/projectfile/projectfile.go#L494

Shouldn't that be enough, or do we still need this line?

Reply:
No, I did not catch that. The depth at which this side-effect is buried is somewhat unnerving. I'll remove the explicit update and test locally to be sure it still has the desired behavior.